MapboxNavigation/NavigationTableViewCell.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 import MapboxDirections
 import MapboxCoreNavigation
 
-class RouteTableViewCell: UITableViewCell {
+class NavigationTableViewCell: UITableViewCell {
```

Comment:
It may not be necessary to replace all instances of “route” with “navigation”. This table is a tabular representation of a single route. If in the future this SDK includes additional navigation UI, like view controllers for roadside search or destination details, this table will still be a tabular representation of a route.